### PR TITLE
feat(pkg): initial release config

### PR DESCRIPTION
**Pull request by esatterwhite**
Initial setup for npm publishing with a shared config, using the config
itself. Automates the Change log updates, github releases, etc based
on conventional commit style messages

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      'close', 'closes', 'closed', 'fix'
+    , 'fixes', 'fixed', 'resolve', 'resolves'
+    , 'resolved', 'ref'
+    ]
+  }
+, writerOpts: {
+    transform: commit.transform
+  , commitPartial: templates.commit
+  }
+, releaseRules: [
```
Comment (mdeltito) [inline index.js:28]: does this need `feat`?

**Review by mdeltito — COMMENTED** [review]
LGTM just a question

File: index.js (hunk 1 above)
Comment (esatterwhite) [reply index.js:28]: I think that an implicit thing, but will add it for the sake of explicitness

**Review by mdeltito — DISMISSED** [review]

**Comment by esatterwhite** [comment]
There are some empty `.gitkeep` files that can be removed

**Review by mdeltito — APPROVED** [review]

**Review by darinspivey — APPROVED** [review]
Thanks, this will be helpful to circle back on our OSS repos and implement, namely for the commitlint sugar

**Review by ligerzero459 — APPROVED** [review]

**Pull request merged**